File: DSA/VenikaPanwar_590013755/Day_8_Question_2.java
import java.util.*;

public class Day_8_Question_2 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        System.out.println("Enter the string:");
        String str = sc.nextLine();

        System.out.println("Enter the character to count:");
        char target = sc.next().charAt(0);

        int count = 0;
        for (int i = 0; i < str.length(); i++) {
            if (str.charAt(i) == target) {
                count++;
            }
        }

        System.out.println("Frequency of '" + target + "': " + count);
    }
}

File: DSA/PranyaGupta_500126032/Day8Question2.py
class Solution:
    def Frequency(self,s, c):
        count = 0
        for char in s:
            if char == c:
                count+=1
        return count        

def main():
    s = input("Enter the String: ")
    c = input("Enter the character: ")
    sol = Solution()
    result = sol.Frequency(s,c)
    print(result)

if __name__ == "__main__":
    main()
